packages/telemetry/src/otel-and-flight-recorder.js

```diff
@@ -0,0 +1,11 @@
+import { makeSlogSender as makeOtelSlogSender } from './otel-trace.js';
+import { makeSlogSender as makeFlightRecorderSlogSender } from './flight-recorder.js';
+
+export const makeSlogSender = async opts => {
+  const flightRecorderSlogSender = await makeFlightRecorderSlogSender(opts);
+  const otelSlogSender = await makeOtelSlogSender(opts);
+  return obj => {
+    flightRecorderSlogSender(obj);
+    return otelSlogSender(obj);
```

Comment:
this looks like a return value of `flightRecorderSlogSender(obj)` is thrown away... but on closer look, it seems that both of these calls have `void` returns; why return anything?